doc/nrf/ug_zigbee.rst

```diff
@@ -27,13 +27,15 @@ It includes all mandatory features of the |zigbee_version| specification and pro
 The stack comes with the following features:
 
 * Complete implementation of the Zigbee core specification and Zigbee Pro feature set.
-* Support for all device roles: coordinator, router, and end device.
-* Zigbee Cluster Library (a specification for end devices).
+* Support for all device roles: Coordinator, Router, and End Device.
+* Zigbee Cluster Library.
 * Base Device Behavior.
 * Devices, described in former Zigbee Home Automation and Light Link profiles.
 * Zigbee Green Power.
```

Comment:
Zigbee Green Power Proxy Basic

It is important to describe this precisely as we support only the mandatory subset.